sklearn/utils/tests/test_graph.py

```diff
@@ -5,8 +5,10 @@
 from scipy import sparse
 
 from sklearn.utils.graph import graph_laplacian
+from sklearn.utils.testing import ignore_warnings
 
 
+@ignore_warnings(category=DeprecationWarning)
 def test_graph_laplacian():
```

Comment:
sure, the very same test is done in scipy directly https://github.com/scipy/scipy/blob/master/scipy/sparse/csgraph/tests/test_graph_laplacian.py#L61

however, maybe we might want to add tests for the extra math properties of graph Laplacians here (e.g. for a fully connected graph, the first eigenvector is constant) in this file